lua/nvim_lsp/elixirls.lua

```diff
@@ -0,0 +1,110 @@
+local configs = require 'nvim_lsp/configs'
+local util = require 'nvim_lsp/util'
+
+local server_name = "elixirls"
+local bin_name = "elixir-ls"
+local bin
+
+local function make_installer()
+    local P = util.path.join
+    local install_dir = P{util.base_install_dir, server_name}
+    local git_dir = P{install_dir, bin_name}
+
+    if vim.fn.has('mac') == 1 or vim.fn.has('unix') == 1 then
+        bin = P{git_dir, "release", "language_server.sh"}
+    elseif vim.fn.has('win32') == 1  or vim.fn.has('win64') == 1 then
+        bin = P{git_dir, "release", "language_server.bat"}
+    else
+        error("System is not supported, try to install manually.")
+        return
+    end
```

Comment:
@mortepau https://github.com/neovim/nvim-lsp/pull/243/files#r427125879
How about this?

```suggestion
local server_name = "elixirls"
local bin_name = "elixir-ls"
local cmd = "language_server"

if vim.fn.has('mac') == 1 or vim.fn.has('unix') == 1 then
    cmd = cmd..".sh"
elseif vim.fn.has('win32') == 1  or vim.fn.has('win64') == 1 then
    cmd = cmd..".bat"
else
    error("System is not supported, try to install manually.")
    return
end

local function make_installer()
    local P = util.path.join
    local install_dir = P{util.base_install_dir, server_name}
	local cmd_path = P{install_dir, bin_name, "release", cmd}
```
and replace bin to cmd_path.

Reply:
@h-michael Yes. That looks cleaner in my opinion.